src/bundle/Resources/views/admin/systeminfo/composer.html.twig

```diff
@@ -2,27 +2,52 @@
 
 <!-- Tab name: {{ 'tab.name.composer'|trans|desc('Composer') }} -->
 
-<h1>{{ 'composer'|trans|desc('Composer') }}</h1>
-<dl>
-    <dt>{{ 'composer.minimum_stability'|trans|desc('Minimum stability') }}</dt>
-    <dd>{{ info.minimumStability }}</dd>
-
-    {% if info.packages is empty %}
-        <dt>{{ 'packages'|trans|desc('Packages') }}</dt>
-        <dd>{{ 'packages.empty'|trans|desc('No packages installed.') }}</dd>
-    {% endif %}
-</dl>
+<table class="table ez-table--list">
+    <thead>
+    <tr>
+        <th colspan="2">
+            {% if info.packages is empty %}
+                {{ 'packages.empty'|trans|desc('No packages installed.') }}
+            {% else %}
+               {{ 'composer'|trans|desc('Composer') }}
+            {% endif %}
+        </th>
+    </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>{{ 'composer.minimum_stability'|trans|desc('Minimum stability') }}</td>
+            <td>{{ info.minimumStability }}</td>
+        </tr>
+    </tbody>
+</table>
 
-{% if info.packages is not empty %}
-    <h2>{{ 'packages'|trans|desc('Packages') }}</h2>
-    <dl>
-        {% for package in info.packages %}
+<div class="ez-table-header round">
+    <h2 class="ez-table-header--label">{{ 'packages'|trans|desc('Packages') }}</h2>
+</div>
+<table class="table">
+    <thead>
+    <tr>
```

Comment:
indentation. It's also valid for other places. Please check indentation in twigs.